donut/modules/feedback/permissions.py

```diff
@@ -0,0 +1,23 @@
+import enum
+
+
+class BOD_PERMISSIONS(enum.IntEnum):
+    # view the summary page (BoD members only)
+    SUMMARY = 8
+    TOGGLE_READ = 9
+    ADD_REMOVE_EMAIL = 10
+    VIEW_EMAILS = 11
+
+
+class ARC_PERMISSIONS(enum.IntEnum):
+    SUMMARY = 12
```

Comment:
I think permission 12 is now used for masquerading, so we should pick a different number. Are you sure these are all in the database? I thought Logan looked for an unused permission number